server/app/views/applicant/ProgramIndexView.java

```diff
@@ -29,6 +29,7 @@
 import services.program.ProgramDefinition;
 import views.BaseHtmlView;
 import views.HtmlBundle;
+import views.components.Icons;
```

Comment:
Top-level comment: I'd consider adding before/after screenshots to the PR description. That'd make understanding the Tailwind style applications a bit easier.

Reply:
Done :)